app/controllers/schedule_controller.rb

```diff
@@ -10,7 +10,11 @@ def save_scheduled_publishing_datetime
       issues = checker.pre_submit_issues
 
       if issues.any?
-        flash[:scheduled_publishing_datetime_issues] = issues.items_for(:scheduled_datetime)
+        href = { scheduled_datetime: "#scheduled_publishing_datetime" }
+        flash["alert_with_items"] = {
+          title: I18n.t!("requirements.scheduled_datetime.title"),
+          items: issues.items_for(:scheduled_datetime, hrefs: href),
```

Comment:
It looks like we want to get all issues here - could we use the `items` method instead?